Lib/test/test_enum.py

```diff
@@ -325,7 +325,10 @@ class IntLogic(int, Enum):
     def test_contains(self):
         Season = self.Season
         self.assertIn(Season.AUTUMN, Season)
-        self.assertNotIn(3, Season)
+        with self.assertRaises(TypeError):
+            3 in Season
+        with self.assertRaises(TypeError):
+            'AUTUMN' in Season
 
```

Comment:
Let's change these to `assertRaisesRegex` and test for the qualname being returned; something like

    self.assertRaisesRegex(TypeError, r"unsupported operand type.s. for .in.: .int. and .Season."):